app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,8 @@
+class RegistrationsController < Devise::RegistrationsController
+
+  protected
+
+    def after_update_path_for(user)
```

Comment:
Unused method argument - `user`. If it's necessary, use `_` or `_user` as an argument name to indicate that it won't be used. You can also write as `after_update_path_for(*)` if you want the method to accept any arguments but don't care about them.